daml-foundations/daml-ghc/daml-stdlib-src/DA/Internal/Template.daml

```diff
@@ -1,6 +1,7 @@
 -- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
 
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
@neil-da I forgot this in the beginning. I hope we can live with that.

Reply:
Sure - it only impacts this one function.